File: TalentHuntService/src/main/java/com/tyss/talenthuntservice/main/binding/Requirement.java
package com.tyss.talenthuntservice.main.binding;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlRootElement;

import lombok.Data;

@Data
@XmlRootElement(name = "REQUIREMENT")
@XmlAccessorType(XmlAccessType.FIELD)
public class Requirement {
	private Integer requirementId;
    private String clientName;
	private Integer noOfPosition;
    private String stack;
    private String desg;
    private Integer expRequired;
    private String jobDesc;
    private String location;
    private String closingDate;
}

File: TalentHunt/src/app/Models/requirement.ts
export class Requirement {

    constructor(
                public requirementId:number,
                public clientName: string,
                public noOfPosition:number,
                public stack: string,
                public desg: string,
                public expRequired: string,
                public jobDesc: string,
                public location: string,
                public closingDate: string  ) {}
}